packages/location-field/src/styled.js

```diff
@@ -83,6 +102,35 @@ export const InputGroupAddon = styled.span`
   text-align: center;
 `;
 
+export const MenuItemA = styled.a`
+  background-color: ${props => (props.active ? "#337ab7" : "transparent")};
+  clear: both;
+  color: ${props => (props.active ? "#fff" : "#333")};
+  display: block;
+  font-weight: 400;
+  line-height: 1.42857143;
+  padding: 3px 20px;
+  text-decoration: none;
+  white-space: nowrap;
+`;
+
+export const MenuItemHeader = styled.li`
+  color: navy;
+  display: block;
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 1.42857143;
+  padding: 3px 20px;
+  text-align: ${props => (props.centeredText ? "center" : "left")};
+  white-space: nowrap;
+`;
+
+export const MenuItemLi = styled.li`
+  &:hover {
+    background-color: ${props => !props.disabled && "#f5f5f5"};
+  }
+`;
+
```

Comment:
Are these components just moved around this file? If so, I don't fully understand why?

Reply:
As part of updating the eslint config, the new rule of [`no-use-before-define`](https://eslint.org/docs/rules/no-use-before-define) became active. The moving around of these variables is to comply with this new rule.